db/seeds.rb

```diff
@@ -32,5 +44,13 @@ def make_email(data)
       }
     )
   )
+
   user.save
+
+  (0..Random.new.rand(5..20)).each do |_i|
+    user.articles.create do |new_article|
+      new_article.message = Faker::Lorem.paragraph_by_chars(number: 120, supplemental: false)
+      new_article.created_at = Faker::Date.between(from: 20.days.ago, to: Date.today)
```

Comment:
In production environment database is usually being set up with `bin/rails db:setup` which means it will execute this code. In your production environment you don't have `Faker` gem so it will fail. So maybe it is better to move this to `db/seeds/development.rb`?

Reply:
don't know about that